Java/Classes.java

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * OOPs (Object-Oriented Programming System)
+ * 
+ * Object means a real-world entity such as a pen, chair, table, computer,
+ * watch, etc.
+ * Object-Oriented Programming is a methodology or paradigm to design a program
+ * using classes and objects.
+ * It simplifies software development and maintenance by providing some
+ * concepts:
+ * 
+ * Object
+ * Class
+ * Inheritance
+ * Polymorphism
+ * Abstraction
+ * Encapsulation
+ * 
+ * Apart from these concepts, there are some other terms which are used in
+ * Object-Oriented design:
+ * 
+ * Coupling
+ * Cohesion
+ * Association
+ * Aggregation
+ * Composition
+ */
+
+class Vehicle {
+    /**
+     * OBJECTS
+     * -----------------------------------------------------
+     * Any entity that has state and behavior is known as an object.
+     * For example, a chair, pen, table, keyboard, bike, etc.
+     * It can be physical or logical.
+     * 
+     * An Object can be defined as an instance of a class.
+     * An object contains an address and takes up some space in memory.
+     * Objects can communicate without
+     * knowing the details of each other's data or code.
+     * The only necessary thing is the type of message accepted and the type of
+     * response returned by the
+     * objects.
+     * 
+     */
+
+    //data members, private in nature
+    private String vehicleNumber;
+    private float price;
+    private String model;
+    private String color;
+    private String variant;
+    private String vehicleName;
+
+    public String getVehicleNumber() {
```

Comment:
- Use these access modifier in the access modifier at the time of that concepts only.
- for now you can use public